rotkehlchen/tests/unit/globaldb/test_globaldb_consistency.py

```diff
@@ -537,6 +546,34 @@ def test_remote_updates_consistency_with_packaged_db(
     if len(missing_unsupported_assets) > 0:
         missing_in_remote_updates.append(missing_unsupported_assets)
 
+    # find all the counterparty_asset_mappings that are present in remote updates but not in packaged db  # noqa: E501
+    missing_in_packaged_db_cp = [
+        f"INSERT INTO counterparty_asset_mappings(local_id, counterparty, symbol) VALUES('{identifier}', '{counterparty}', '{symbol}');"  # noqa: E501
+        for (counterparty, symbol), identifier in updated_counterparty_asset_mappings.items()
+        if (counterparty, symbol) not in packaged_counterparty_asset_mappings
+    ]
+
+    # find all the counterparty_asset_mappings that are present in packaged db but not in remote updates  # noqa: E501
+    missing_in_remote_updates_cp = [
+        {
+            'asset': identifier,
+            'counterparty': counterparty,
+            'counterparty_symbol': symbol,
+        }
+        for (counterparty, symbol), identifier in packaged_counterparty_asset_mappings.items()
+        if (counterparty, symbol) not in updated_counterparty_asset_mappings
+    ]
+
+    # find all the counterparty_asset_mappings that are present in both but have different values  # noqa: E501
+    missing_in_packaged_db_cp.extend([
+        f'UPDATE counterparty_asset_mappings SET local_id = "{updated_counterparty_asset_mappings[counterparty, symbol]}" WHERE counterparty = "{counterparty}" AND symbol = "{symbol}";'  # noqa: E501
+        for (counterparty, symbol), identifier in packaged_counterparty_asset_mappings.items()
+        if (counterparty, symbol) in updated_counterparty_asset_mappings and updated_counterparty_asset_mappings[counterparty, symbol] != identifier  # noqa: E501
+    ])
```

Comment:
is it possible to further abstract this with the locations one? The logic looks quite similar